src/ukify/ukify.py

```diff
@@ -1314,16 +1318,16 @@ def make_uki(opts: UkifyConfig) -> None:
                 continue
 
             print(
-                f"Copying section '{n}' from '{profile}': {pesection.Misc_VirtualSize} bytes",
+                f"Copying section '{n}' from '{profile}': {pe_section_size(pesection)} bytes",
                 file=sys.stderr,
             )
             uki.add_section(
-                Section.create(n, pesection.get_data(length=pesection.Misc_VirtualSize), measure=True)
+                Section.create(n, pesection.get_data(length=pe_section_size(pesection)), measure=True)
```

Comment:
Isn't this the same as doing `pesection.get_data(ignore_padding=True)`?

Reply:
Too new, we can't rely on it yet

Reply:
It seems to have been added in commit [0db5deb5871e6e5968bc23356bcdf704483ce2e6](https://github.com/erocarrera/pefile/commit/0db5deb5871e6e5968bc23356bcdf704483ce2e6), though this was only [merged](https://github.com/erocarrera/pefile/commit/c9e8c59b56cfa14dbeca8cb9e33d3d84056102a6) after 2021.9.3 was released, so 2022.5.30 is the first release including it, which excludes Debian bullseye as well as Ubuntu focal and jammy, [looking at pkgs.org](https://pkgs.org/search/?q=pefile). They're all pretty old, do they still get backports of recent ukify?

Reply:
No but ukify might be built locally, it doesn't feel worth it to bump the version requirement for what's essentially one line of code.